packages/zowe-explorer-api/src/utils/files.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+export const PERM_VALUES = {
+    r: 4,
+    w: 2,
+    x: 1,
+};
+
+export type FileAttributes = {
+    gid: number;
+    group: string;
+    owner: string;
+    uid: number;
+    perms: string;
+};
+
+export function permStringToOctal(perms: string): number {
```

Comment:
Thanks for writing your own method instead of looking for a package to do this for us.
I know we have gone back and forth on this, but (personally) I rather implement utility methods like this one instead of depending on some third-party package. 😋